### PR TITLE
#6 create basic project structure

### PR DESCRIPTION
**Pull request by AVM82**
connect to #6

### CODE REVIEW

**Review by Shtramak — APPROVED** [review]
Your first commit has name not according to requirements, there should be "#6: " at the begining like in other commits. I had the same issue) And also, as for me, it's ok to exclude "workflow.md" from our branches cause this file does not concern to a project...

File: src/test/java/ApplicationTest.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import juja.sqlcmd.Application;
```
Comment (Shtramak) [inline src/test/java/ApplicationTest.java]: It looks like you have different structure of "main" and "test" folders... It is good practice this folders to have the same structure, so test classes can access (and test!) package-scope classes and members. Here's a good description of this point - http://bit.ly/2hyE8uO

**Review by Shtramak — APPROVED** [review]

File: src/test/java/ApplicationTest.java (hunk 1 above)
Comment (AVM82) [reply src/test/java/ApplicationTest.java]: I never attached importance to this, but you are right. 
Thanks for link.
I'll changed structure of "test" folders.

**Review by kortov — APPROVED** [review]
Интересно в pom прописал конфиги для компилятора

**Review by ivshapovalov — APPROVED** [review]
Good

**Review by DenysMelnychenko — APPROVED** [review]

**Review by adaleorion — APPROVED** [review]
thanks, well done 👍

**Pull request merged**